foyer/atomtyper.py

```diff
@@ -1,20 +1,20 @@
 from warnings import warn
 
-import parmed as pmd
+from gmso import Topology
+from parmed import Structure
 import parmed.periodic_table as pt
 
 from foyer.exceptions import FoyerError
 from foyer.topology_graph import TopologyGraph
 from foyer.smarts_graph import SMARTSGraph
 
-
 def find_atomtypes(structure, forcefield, max_iter=10):
     """Determine atomtypes for all atoms.
 
     Parameters
     ----------
-    topology : parmed.Structure or TopologyGraph
-        The topology that we are trying to atomtype.
+    topology : parmed.Structure, or gmso.Topology, or TopologyGraph
+        The topology that we are trying to atomtype, must matched with backend.
```

Comment:
This does not tell the user what backend options are available. This should be explained a bit more.